backend/TreeOfAKind.Infrastructure/Domain/Trees/TreeEntityTypeConfiguration.cs

```diff
@@ -18,22 +18,61 @@ public void Configure(EntityTypeBuilder<Tree> builder)
             builder.HasKey(t => t.Id);
 
             builder.Property(t => t.Name)
-                .HasMaxLength(StringLengths.VeryShort);
+                .HasMaxLength(StringLengths.Short);
 
             builder.OwnsMany(t => t.TreeOwners, b =>
             {
                 b.ToTable("TreeUserProfile", SchemaNames.Trees);
                 b.HasKey(to => new {to.TreeId, to.UserId});
             });
 
-            builder.OwnsMany<Person>(t => t.People, b =>
+            builder.OwnsMany(t => t.People, b =>
             {
                 b.ToTable("People", SchemaNames.Trees);
 
                 b.HasKey(p => p.Id);
 
                 b.WithOwner(p => p.Tree);
+
+                b.Property(p => p.Name)
+                    .HasMaxLength(StringLengths.VeryShort);
+
+                b.Property(p => p.Surname)
+                    .HasMaxLength(StringLengths.VeryShort);
+
+                b.Property(p => p.Description)
+                    .HasMaxLength(StringLengths.Short);
+
+                b.Property(p => p.Biography)
+                    .HasMaxLength(StringLengths.VeryLong);
+
+                b.Property(p => p.BirthDate)
+                    .HasColumnType("date");
+
+                b.Property(p => p.DeathDate)
+                    .HasColumnType("date");
+
+                b.Property(p => p.Gender)
+                    .HasConversion<string>()
```

Comment:
Why?

Reply:
Cause I hate enum to int and enum to tale conversions